code/src/components/Product.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+//this component Product, whill show the product when we fetched it
+import styled from 'styled-components'
+
+const ProductContainer = styled.div`
+    background:  #e5faf5;
+    background-image: url('../assets/tag.jpg');
+`
+const ProductHeading = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    color: darkslategrey;
+`
+const ProductInfo = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    color: darkslategrey;
+`
+const ProductStore = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    color: darkslategrey;
+`
+const ProductImg = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+`
+//this component Product, whill show the product when we fetched it
+export const Product = () => {
+    const scan = useSelector(state => state.products.product)
+    // if we dont have a product at all, we dont want to show anything, return null
+    if (!scan) return null
+    // if we get a product, we want to show it
+    console.log('SCAN', scan)
+    return (
+        <ProductContainer>
+            <ProductHeading>
+                <img src={scan.product && scan.product.image_thumb_url} alt="" />
+                <h1>{scan.product && scan.product.product_name}</h1>
+                {/* <h4>Ingredients:{product.product && product.product.ingredients_tags}</h4> */}
+            </ProductHeading>
+            <ProductInfo>
+                <h4>Ingredients:</h4>
+                <h4>{scan.product && scan.product.ingredients_tags}</h4>
```

Comment:
Maybe here you could try to get rid of the en: part of the ingredients. To make it more readable.
This is what I get scanning a yoghurt
Ingredients:en:whole-milken:dairyen:milken:creames:leche-en-polvo-desnatadaes:fermentos-lacteos
So maybe to space it out a bit and try to remove whatever comes before the : 
Check here to get some hints.
https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Useful_string_methods